### ISSUE TITLE
polsatnews.pl

### ISSUE THREAD

**Issue by Barticf** [bogachenko/fuckfuckadblock#162]
### Basic information

**Check the link:** `https://www.polsatnews.pl/wiadomosc/2021-03-22/te-produkty-znikna-z-polek-komisja-europejska-walczy-z-tluszczami-trans/`
**Category:** Anti-adblock
**Reported from:** Poland

### User environment

**Browser:** Firefox
**Extension or program:** uBlock

### Additional message

No video when Fuck Fuckadblock is active

**Comment by bogachenko** [comment]
I can reproduce the problem only if I block the domain `imasdk.googleapis.com`, but it is unblocked by the rule, so I don't have an anti-block.

https://github.com/bogachenko/fuckfuckadblock/blob/fd717ed4b79ae4d7acb6088d4781aeb6cb746aab/fuckfuckadblock.txt#L445

And by the way, please use old issues to ping problem sites.
***
Duplicate of #150

**Comment by bogachenko** [comment]
The problem was on my side, the browser based on the chrome engine on my computer can't reproduce this problem as soon as I changed the browser - boom.
